core/src/main/java/org/apache/iceberg/SchemaParser.java

```diff
@@ -277,6 +313,17 @@ private static Types.MapType mapFromJson(JsonNode json) {
     }
   }
 
+  private static Types.GeometryType geometryFromJson(JsonNode json) {
+    String crs = JsonUtil.getStringOrNull("crs", json);
```

Comment:
nit: these (and following) can be replaced by the constants?

Reply:
This is reverted since we are not encoding geospatial types as JSON.